index.html

```diff
@@ -181,7 +180,7 @@ <h1 class="title is-1 pb-6">
         <div class="container">
             <h1 class="title is-1"><a name="showcase">#</a> Showcase</h1>
             <br>
-            <p class="subtitle is-4">Made something cool with <b>wgpu-rs</b>? Make a PR to <a href="https://github.com/gfx-rs/wgpu-rs.github.io">wgpu-rs.github.io</a>, and reach out to us on <a href="https://matrix.to/#/#wgpu-users:matrix.org">#wgpu-users</a>!</p>
+            <p class="subtitle is-4">Made something cool with <b>wgpu</b>? <a href="https://github.com/gfx-rs/wgpu-rs.github.io"></a>Make a PR</a>, and reach out to us on <a href="https://matrix.to/#/#wgpu-users:matrix.org">#wgpu-users</a>!</p>
```

Comment:
@kvark I made a mistake and let a `</a>` slip in. Can you push a quick fix to delete that stray closing tag?

Reply:
done